### ISSUE TITLE
Florida Straits transport analysis

### ISSUE THREAD

**Issue by atb299** [NOC-MSM/RAPID-Evolution#3]
Florida Straits transport is significantly weaker than observed:

![Image](https://github.com/user-attachments/assets/67c28c9b-a5aa-4445-a566-f2dc2c2a189f)

- [x] Compute Florida Straits transports for all simulations: NPD, R-Evo1, GO8p7{h,m}, SE-NEMO
- [x] Analyse velocities - is transport moving east of Bahamas, or is it weakening?
- [ ] Fix the problem

**Comment by atb299** [comment]
Velocity through FS from the NPD eORCA025 simulation:
![Image](https://github.com/user-attachments/assets/e76739dd-e8bb-4c27-9a8b-fcd5b8317c04)

**Comment by atb299** [comment]
There's a T and S change with vertical structure that appears in the NPD simulation:

![Image](https://github.com/user-attachments/assets/76acfe42-79a4-4692-8da7-cfdba9c23628)
![Image](https://github.com/user-attachments/assets/e1cb1046-df63-4570-a946-4aad55767012)

I expect I'll find similar changes in the other simulations.

**Comment by jdha** [comment]
![Screenshot 2024-12-09 at 21 54 23](https://github.com/user-attachments/assets/6a5d8f62-c50e-4c2f-b05d-17fdbe540fed)

**Comment by jdha** [comment]
![Screenshot 2024-12-09 at 22 16 17](https://github.com/user-attachments/assets/7ee0fc63-f98e-4599-ae40-2a4b13ae34af)

I'll do this properly later - but you can see that it's similar to the NPD structure

**Comment by jbkajtar** [comment]
Here are some findings for eORCA025-RAPID12-RAPID36:
![so_abs](https://github.com/user-attachments/assets/f53e9d89-8bef-4f17-8604-cbe89bfd1fa7)
![theta_con](https://github.com/user-attachments/assets/d91c675d-5382-4f0f-9b45-f67d4e376d32)
![vo](https://github.com/user-attachments/assets/004df586-cd4a-4503-96a5-e3ad54d4cb6e)

Edit: updated depth coordinate for vo

**Comment by atb299** [comment]
It seems that the FS and Antilles transports both reduce in the models we've looked at. The DWBC seems to increase in the coarser (1/4) resolution models. All the models have the same common origin and forcing, so I guess the similarity is no real surprise. So I guess the question is what causes the reduction in the FS and Antilles transports? Is it drift in the T and S, or due to the forcing, or something else?

I found https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2018jc014836, which looks like a good description of the Antilles current in the last ~20 years.

**Comment by bablusinha** [comment]
yes don't know if you remember I compared the Antilles transports in the 1/36 model with the Meinen paper and it gets the structure and the variability quite well. So it seems to be the magnitude of the current that is mostly affected.

**Comment by bablusinha** [comment]
ps these pictures are very illuminating!

**Comment by atb299** [comment]
T, S and rho anomalies from the NPD eORCA025 simulation, plus same zoomed into the FS. I don't know why Metric gives rho on a different grid - need to look into that...

<img width="798" alt="Screenshot 2024-12-10 at 10 11 38" src="https://github.com/user-attachments/assets/e46d90c3-9883-47d3-8350-216c8eeded06">
<img width="805" alt="Screenshot 2024-12-10 at 10 15 22" src="https://github.com/user-attachments/assets/acbf1360-7f88-4075-937c-b506319cb7c1">

**Comment by atb299** [comment]
Looks like model forcing is a key controller here:

<img width="944" alt="Image" src="https://github.com/user-attachments/assets/5e29f567-6895-424c-9f7d-d8e2daeafab6" />

It still drops a bit after ~2004, but mainly in the FS. The change in the Antilles and DWBC are much smaller than in the JRA55-do forced simulation:

**ERA5**
<img width="814" alt="Image" src="https://github.com/user-attachments/assets/073363e5-8efa-4091-90d9-6b6bee69e7c8" />

**JRA55**
<img width="810" alt="Image" src="https://github.com/user-attachments/assets/6a19a3a7-fa11-4e40-898c-7b9bb3c429e4" />
